router/v3/channels_test.go

```diff
@@ -65,6 +66,29 @@ func channelEquals(t *testing.T, expect *model.Channel, actual *httpexpect.Objec
 	actual.Value("children").Array().ContainsOnly(childIDs...)
 }
 
+func channelListElementEquals(t *testing.T, expect []*model.Channel, actual *httpexpect.Array) {
+	t.Helper()
+	// copy to avoid modifying `expect`
+	expectCopy := make([]*model.Channel, len(expect))
+	copy(expectCopy, expect)
+
+	channelCount := int(actual.Length().IsEqual(len(expect)).Raw())
+	// do not use `expect`, use `expectCopy` instead
+	for i := 0; i < channelCount; i++ {
```

Comment:
エディタにおすすめされたんだけど、最近はこんな書き方もできるようです
ただ、動作上の問題はないのでこれを適用するかはお任せします！
```suggestion
	for i := range channelCount {
```